api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -650,7 +650,7 @@ public OrderType getOrderTypeByUuid(String uuid) {
 
 	@Override
 	public OrderType getOrderTypeByClassName(String javaClassName) {
-		return (OrderType) sessionFactory.getCurrentSession().createQuery("from OrderType o where o.javaClassName = :javaClassName")
+		return (OrderType) sessionFactory.getCurrentSession().createQuery("from OrderType o where o.javaClassName = :javaClassName", OrderType.class)
```

Comment:
We should be able to get rid of the cast here, right?

Reply:
Let me see how to get around it with no conflict.